content/docs/develop/reference/bridge/version_check/index.md

```diff
@@ -0,0 +1,54 @@
+---
+title: Daily Version Check
```

Comment:
Can we generalize this for Bridge *and* CLI? Or should we add an extra page for the CLI?

Reply:
Let's add an extra page for the CLI because enabling/disabling the version check is different. As mentioned in the last dev meeting, we need to streamline the messaging for the CLI version check with the Bridge version check. I will create an issue for that.

Reply:
Thanks!